tooling/nargo_fmt/src/config.rs

```diff
@@ -44,20 +44,25 @@ macro_rules! config {
 config! {
     tab_spaces: usize, 4, "Number of spaces per tab";
     remove_nested_parens: bool, true, "Remove nested parens";
+    error_on_unformatted: bool, false, "Error if unable to get comments";
```

Comment:
The comment here doesn't match up with the flag name. From the flag name, it sounds like a `-Werror` equivalent but the comment suggests it has something to do with comments.

Reply:
something like `error_on_lost_comment`?

Reply:
Sure - and shouldn't a lost comment be an error by default?

Reply:
by default, we replace it with the original expression; perhaps we should change this behavior

Reply:
I don't think we should ever ideally lose or rewrite a user's comment so if it does happen I think an error would be preferable